app/src/main/java/com/simongellis/vvb/menu/VideoMenuFragment.kt

```diff
@@ -92,6 +97,51 @@ class VideoMenuFragment: PreferenceFragmentCompat() {
             true
         }
 
+        findPref(Prefs.COLOR).setOnPreferenceChangeListener { _, newColor ->
+            val newColor = newColor as Int
+            val oldColor = _sharedPreferences.getInt(Prefs.COLOR.prefName, Color.RED)
+            if(newColor === 0){
+                _customColorPicker = AmbilWarnaDialog(
```

Comment:
Just as a sanity check, if you rotate your screen while this color picker is open, does it keep working or do you get a crash? I remember running into issues with that early on because I wasn't registering dialogs with a fragment manager or something

Reply:
Looking at [the readme](https://github.com/yukuku/ambilwarna#how-to-use-it-as-a-preference), this library comes with a preference that may be able to do some of the dialog management itself

Reply:
ill def test this and get back to you

Reply:
It looks like it doesn't crash on my phone, just closes the dialog. That's probably fine, though I'll probably come back and use a single color picker later on.